src/Plugin/os2web/DataLookup/ServiceplatformenBase.php

```diff
@@ -281,7 +285,7 @@ protected function query($method, array $request) {
     if (!$this->isReady()) {
       return [
         'status' => FALSE,
-        'text' => $this->getStatus(),
+        'error' => $this->getStatus(),
```

Comment:
i don't get where this change is coming from. According to GIT history it was always like "error" array key.
Anyways it's used as "error" key everywhere, so that is correct.

Reply:
It has been `'text'` for 4 years (cf. https://github.com/OS2web/os2web_datalookup/blame/bbe14aecb303b5b8d3196eb20730e72b7151956b/src/Plugin/os2web/DataLookup/ServiceplatformenBase.php#L284).

Reply:
you're absolutely right, i was looking at the wrong place